corehq/apps/events/models.py

```diff
@@ -88,7 +88,8 @@ class Event(models.Model):
 
     name = models.CharField(max_length=100)
     domain = models.CharField(max_length=255)
-    event_id = models.CharField(max_length=255, unique=True)
+    event_id = models.UUIDField(primary_key=True, default=uuid.uuid4)
+    _case_id = models.UUIDField(null=False, default=uuid.uuid4)
```

Comment:
Since we're setting all the id fields as UUID, should we do the same for `attendance_taker_ids`? We'll will just need to be careful if doing this as we'll need to make sure to convert the ids to string and remove the hyphens when using it in the attendance taker list.

Reply:
c191ae8e4b1